libs/pxweb2-ui/src/lib/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,81 @@
+//Adapted from: Juliana Godoy Viana, a11ytabs (Original copyright notice included in copyright_notice.txt)
```

Comment:
We should probably mention somewhere here that it is released with a MIT license